src/table/matrix_table.cpp

```diff
@@ -90,6 +90,19 @@ void MatrixWorkerTable<T>::Get(const std::vector<integer_t>& row_ids,
 }
 
 template <typename T>
+void MatrixWorkerTable<T>::Get(T* data, size_t size, integer_t* row_ids,
```

Comment:
No, I think the size should be the size of the 'data' array. This is implemented by you or Xiao? You need to make sure with him.